src/test/java/integration/Config.java

```diff
@@ -88,4 +88,19 @@ void imageDefault() throws IOException {
         imageDefaultOutput,
         CoreMatchers.containsString(versionOutput.trim()));
   }
+
+  @Test
```

Comment:
Can you please also add a new method to the `unit.Config` test to check that your property is getting set and returned properly (like the `appLaunch()` method but for `format`)?

Reply:
done